daml-lf/interface/src/main/scala/com/digitalasset/daml/lf/iface/Interface.scala

```diff
@@ -79,17 +79,30 @@ final case class Interface(
     * }}}
     */
   def resolveChoices(
-      findInterface: PartialFunction[Ref.TypeConName, DefInterface.FWT]
+      findInterface: PartialFunction[Ref.TypeConName, DefInterface.FWT],
+      failIfUnresolvedChoicesLeft: Boolean = false,
```

Comment:
I would prefer an extra method, `resolveChoicesFailUnresolved` or another terrible name :-) over a boolean flag

Reply:
I'm not entirely sure why we all seem to dislike single bool flags so much, however I can fulfill that request.

Reply:
It's a minor nitpick maybe but I do think it helps with the readability of the code, especially if someone would use it without naming the argument. (then you have to go look, what is this `true`)

Reply:
Can you make this change?

Reply:
Completely forgot about this one. I will make that change :)

Reply:
done :)